concent_api/gatekeeper/views.py

```diff
@@ -0,0 +1,72 @@
+from base64                         import b64decode
+
+from django.conf                    import settings
+from django.http                    import JsonResponse, HttpResponse
+from django.views.decorators.http   import require_POST, require_GET
+from django.views.decorators.csrf   import csrf_exempt
```

Comment:
Please split into multiple lines and also try to keep alphabetical.

Reply:
done